tests/orderly/examples/parameters/parameters.py

```diff
@@ -1,8 +1,5 @@
 import orderly
 
-# TODO(mrc-5255): We'll rethink this strategy soon
-# ruff: noqa: F821
-
-orderly.parameters(a=1, b=None)
+parameters = orderly.parameters(a=1, b=None)
 with open("result.txt", "w") as f:
-    f.write(f"a: {a}\nb: {b}\n")  # type: ignore
+    f.write(f"a: {parameters['a']}\nb: {parameters['b']}\n")
```

Comment:
Add some sugar to allow `parameters.a` perhaps?

That (or anything slightly more sophisticated than just using a dict for the storage) would also allow nicer error messages on near miss parameter usage, so that `parameters['whatever']` or `parameters.whatever` might return an error that lists known parameters

Reply:
I've replaced it with a `SimpleNamespace` subclass, which has `.foo` lookup. The subclass is empty for now (other than the inherited init/eq/repr methods), but we can extend it in the future.